docs/tutorials/quickstart.md

```diff
@@ -1,8 +1,194 @@
+* JAX programs support efficient evaluation of gradients via its automatic differentiation transformations.
+* JAX functions can be automatically vectorized to efficiently map them over arrays representing batches of inputs.
+
+## Installation
+
+JAX can be installed for CPU on Linux, Windows, and OSX directly from the [Python Package Index](https://pypi.org/project/jax/):
+```
+pip install "jax[cpu]"
+```
+For more detailed installation information, including installation with GPU support, see {ref}`installation`.
+
+## JAX as NumPy
+
+Most JAX usage is through the familiar {mod}`jax.numpy` API, which is typically imported under the `jnp` alias:
+
+```{code-cell}
+import jax.numpy as jnp
+```
+
+With this import, you can immediately use JAX in a similar manner to typical NumPy programs,
+including using NumPy-style array creation functions, Python functions and operators, and
+array attributes and methods:
+
+```{code-cell}
+def selu(x, alpha=1.67, lmbda=1.05):
+  return lmbda * jnp.where(x > 0, x, alpha * jnp.exp(x) - alpha)
+
+x = jnp.arange(5.0)
+print(selu(x))
+```
+
+You'll find a few differences between JAX arrays and NumPy arrays once you begin digging-in;
+these are explored in  [🔪 JAX - The Sharp Bits 🔪](https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html).
+
+## Using {func}`~jax.jit` to speed up functions
+JAX runs transparently on the GPU or TPU (falling back to CPU if you don't have one). However, in the above example, JAX is dispatching kernels to the chip one operation at a time. If we have a sequence of operations, we can use the `@jit` decorator to compile multiple operations together using XLA.
+
+We can use IPython's `%timeit` to quickly benchmark our `selu` function, using `block_until_ready()` to
+account for JAX's dynamic dispatch (See {ref}`async-dispatch`):
+
+```{code-cell}
+from jax import random
+
+key = random.key(1701)
+x = random.normal(key, (1_000_000,))
+%timeit selu(x).block_until_ready()
+```
+
+(notice we've used {mod}`jax.random` to generate some random numbers; for details on how to generate random
+numbers in JAX, see {ref}`pseudorandom-numbers`).
+
+We can speed the execution of this function with `@jit`, which will jit-compile the first time `selu` is called and will be cached thereafter.
+
+```{code-cell}
+from jax import jit
+
+selu_jit = jit(selu)
+_ = selu_jit(x)  # compiles on first call
+%timeit selu_jit(x).block_until_ready()
+```
+
+The above timing represent execution on CPU, but the same code can be run on GPU or TPU for
+an even greater speedup.
+
+For more on JIT compilation in JAX, see {ref}`jit-compilation`.
+
+## Taking derivatives with {func}`~jax.grad`
+
+In addition to evaluating numerical functions, we also want to transform them.
+One transformation is [automatic differentiation](https://en.wikipedia.org/wiki/Automatic_differentiation).
```

Comment:
```suggestion
One transformation is [automatic differentiation (autodiff)](https://en.wikipedia.org/wiki/Automatic_differentiation).
```

Since we're using the shortened version later.